hscommon/hsfiletypes/filetype.go

```diff
@@ -35,13 +36,17 @@ func determineTBLtype(data *[]byte) (FileType, error) {
 		return FileTypeTBLStringTable, err
 	}
 
-	return FileTypeUnknown, errors.New("unknown file type")
+	d := *data
+	if string(d[:4]) == "Woo!" {
```

Comment:
Is this debug/test stuff, or do the files actually say "Woo!" ?

Reply:
It actually says "Woo!".